.github/workflows/ci-documentbot.yml

```diff
@@ -46,7 +50,7 @@ jobs:
           go-version: 1.18
 
       - name: Labeling
-        uses: apache/pulsar-test-infra/docbot@master
+        uses: ./docbot
```

Comment:
Why we cannot use `apache/pulsar-test-infra/docbot@master` in this case like pulsarbot?

cc @maxsxu

Reply:
@tisonkun I've explained that in this comment: https://github.com/apache/pulsar/pull/16060#issuecomment-1155359543

Reply:
If we adopt docker container action as described in https://github.com/apache/pulsar-test-infra/issues/58, we may need not to checkout the code as well as setup-go, just use the action itself ><